sites/all/themes/cetaviano/sass/partials/components/_home-techs-intro.scss

```diff
@@ -63,77 +64,32 @@
     }
 }
 
-.container-title-course h2 {
+.container-title-course h3 {
```

Comment:
es posible ponerle una clase al h3? asi quitamos el selector decendiente

Reply:
Si, aplico bem o no?
.courses_title

Reply:
Estaba poniendo la clase y no la quiere agarrar, creo que por eso le llegaron así.

Reply:
si, BEM all the way. Como que no la quiere agarrar? desde donde la incluyen, panels? Las Marias son expertas, pediles ayuda